pkg/scheduler/cache/cache.go

```diff
@@ -295,7 +295,25 @@ func (su *defaultStatusUpdater) UpdatePodGroup(pg *schedulingapi.PodGroup) (*sch
 		return nil, err
 	}
 
-	updated, err := su.vcclient.SchedulingV1beta1().PodGroups(podgroup.Namespace).Update(context.TODO(), podgroup, metav1.UpdateOptions{})
+	updated := podgroup.DeepCopy()
```

Comment:
Does this deep copy need？

Reply:
indeed, it can be a var, if err returned by RetryOnConflict is nil, It must not be empty.

Reply:
updated.